### ISSUE TITLE
For the languages problem.

### ISSUE THREAD

**Issue by Lufims** [ligi/SurvivalManual#148]
I have an idea for the languages problem, if the language is an archive, other can translate, and the app could take the archive of other language from somewhere ( i dont how to do this) and erase the archive of the old language. This will not cost much memory.

**Comment by norway-yv** [comment]
Yes, ElevenClock has an exellent solution using Python dictionaries (easy to edit) as language files. Example of what to do (pseudocode):
import languagePacks
language = inputbutton What is your language
language = languagePacks.dictionary(language)
applyLanguageToText()
system.mkdir Languages
system.createfile English.py
English.py.contents = languagePacks.dictonary(en)
Same for chosen language
system.removefile LanguagePacks

**Comment by h8nor** [comment]
Different languages have their own characteristics, so you can’t just replace sentences.
It is best to [break text into blocks](https://github.com/k88hudson/git-flight-rules/pull/357), then both changes and new blocks will be visible.
